packages/web-components/src/components/va-date/test/va-date.e2e.ts

```diff
@@ -0,0 +1,147 @@
+import { newE2EPage } from '@stencil/core/testing';
+import { axeCheck } from '../../../testing/test-helpers';
+
+describe('va-date', () => {
```

Comment:
Can we also get a test showing that the event target's `value` has the updated date for the `dateChange` event?

Reply:
I added to the test where I check for updates: https://github.com/department-of-veterans-affairs/component-library/pull/347/commits/601be1531e3ee837c5397954467269da578a0613#diff-999e0d6b4f145596771d22dcd3d694a5363c5f85675d8a905c4987e89c1cf631

Reply:
I still don't think we have a test for `dateChange` - I think it would be good to have one to test that it's firing when any 1 of the three components has a change.